src/traffic_cop/traffic_cop_util.cpp

```diff
@@ -10,6 +10,8 @@
 #include "optimizer/query_to_operator_transformer.h"
 #include "optimizer/statistics/stats_storage.h"
 #include "parser/drop_statement.h"
+#include "parser/expression/column_value_expression.h"
+#include "parser/expression/constant_value_expression.h"
```

Comment:
Why do you need to include these files here?

Reply:
I've deleted these lines.